tests/Pest.php

```diff
@@ -1,9 +1,14 @@
 <?php
 
+use Tests\CustomTestCase\CustomTestCase;
 use Tests\CustomTestCaseInSubFolders\SubFolder\SubFolder\CustomTestCaseInSubFolder;
 
 uses(CustomTestCaseInSubFolder::class)->in('PHPUnit/CustomTestCaseInSubFolders/SubFolder/SubFolder');
 
+uses(CustomTestCase::class)->in('PHPUnit/CustomTestCase/SubFolder/*/');
```

Comment:
I feel like maybe these should be somewhere else with a more logical name (something like, `GlobPatternTests`) just so we know why the tests are there.

Perhaps add a comment to the test cases too?